fixtures/dom/src/components/fixtures/error-handling/index.js

```diff
@@ -69,7 +104,11 @@ export default class ErrorHandlingTestCases extends React.Component {
             should be replaced with "Captured an error: Oops!" Clicking reset
             should reset the test case.
           </TestCase.ExpectedResult>
-          <Example error={new Error('Oops!')} />
+          <Example
+            throws={() => {
+              new Error('Oops!');
```

Comment:
Actually, given the name, it should be:
```js
throws={() => {
  throw new Error('Oops!');
}}
```
And then we should just remove the `throw` from `BadRenderer`